ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,3 +1195,57 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  hidden: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-ci-secret-bootstrap
+  spec:
+    containers:
+    - args:
+      - --bw-password-path=/etc/ci-secret-bootstrap/bw_password
+      - --bw-user=tbd
+      - --config=./core-services/ci-secret-bootstrap/_config.yaml
+      - --dry-run=false
+      - --force=true
```

Comment:
Do we really wanna do this?

Reply:
Yes, otherwise it is not going to update secrets when they change in BW